integration_tests/basic_client_test.py

```diff
@@ -1,80 +1,60 @@
-import asyncio
-import dotenv
-import logging
-import os
-import sys
-from pyracing import client as pyracing
-from pyracing import constants
-from pyracing.response_objects import career_stats, chart_data, iracing_data, \
-    upcoming_events, historical_data, session_data
-import unittest
-
-dotenv.load_dotenv()
-client = pyracing.Client(
-    os.getenv('IRACING_USERNAME'),
-    os.getenv('IRACING_PASSWORD')
+from integration_tests import (
+    IRacingIntegrationTest,
+    async_test,
 )
-
-logging.basicConfig(
-    stream=sys.stdout,
-    level=logging.INFO,
-    datefmt="%Y-%m-%d %H:%M:%S",
-    format="%(asctime)s;%(levelname)s;%(message)s"
+from pyracing import constants
+from pyracing.response_objects import (
+    career_stats,
+    chart_data,
+    historical_data,
+    iracing_data,
+    session_data,
+    upcoming_events,
 )
 
 
-def async_test(f):
-    def wrapper(*args, **kwargs):
-        coro = asyncio.coroutine(f)
-        future = coro(*args, **kwargs)
-        loop = asyncio.get_event_loop()
-        loop.run_until_complete(future)
-
-    return wrapper
-
-
-class ClientTest(unittest.TestCase):
+class ClientTest(IRacingIntegrationTest):
```

Comment:
Do we need to explicitly call the setUp method? Or is that handled implicitly by the tests? I just want to make sure `self.client` isn't null throughout the tests.